src/shared/utils/isSystemItem.ts

```diff
@@ -0,0 +1,4 @@
+import { LookupTableItems } from 'shared/consts';
+
+export const isSystemItem = (name: string) =>
```

Comment:
could please remove `isSystemItem` from src/modules/Builder/features/ActivitySettings/SubscalesConfiguration/SubscalesConfiguration.utils.tsx
and use only from one place?

Reply:
it has been already removed

Reply:
yeap, I missed it, apparently my eyes are tired at the end of the working day :)